edwardpizarro-fatima.js

```diff
@@ -0,0 +1,222 @@
+//Question 1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+//console.log(calculateCube(5));
+//=> 125
+
+function calculateCube(){
+    console.log(5*25)
+  }
+   calculateCube()
+  //I wasnt sure about this solution, but it came out to 125. Im pretty sure there is a right way of doing this just not sure what it is.
+  
+  
+  //Question 2
+  //Is a Vowel?
+  // Write a function isAVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+  
+  let vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']
+  const isAVowel = (letter) => {
+    if (vowels.indexOf(letter.toUpperCase()) !== -1) {
+      return true
+    }
+    return false
+  }
+  console.log(isAVowel('a'))
+  //Got some help from Seth and Trez on this one. On a scale from 1-10 id say im at a 5 with this one for i dont 100% understand it.
```

Comment:
nice work here! one way to make your code a bit more readable is to create the variable `vowels` inside the function, in the local scope.